Mage.Sets/src/mage/cards/a/AmyPond.java

```diff
@@ -0,0 +1,97 @@
+package mage.cards.a;
+
+import java.util.UUID;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.DealsCombatDamageToAPlayerTriggeredAbility;
+import mage.abilities.dynamicvalue.DynamicValue;
+import mage.abilities.dynamicvalue.common.SavedDamageValue;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.keyword.PartnerWithAbility;
+import mage.cards.*;
+import mage.constants.*;
+import mage.abilities.keyword.DoctorsCompanionAbility;
+import mage.counters.CounterType;
+import mage.filter.common.FilterSuspendedCard;
+import mage.game.Game;
+import mage.players.Player;
+import mage.target.common.TargetCardInExile;
+
+/**
+ *
+ * @author Skiwkr
+ */
+public final class AmyPond extends CardImpl {
+
+    protected static final FilterSuspendedCard filter = new FilterSuspendedCard("suspended card you own");
```

Comment:
`private` appropriate here (this filter is declared in this class and shouldn't be getting used anywhere else)